src/parser/jobs/blm/FireCounter.js

```diff
@@ -65,7 +69,36 @@ export default class FireCounter extends Module {
 	}
 
 	_onComplete() {
+		this._lastStop = true
 		this._stopRecording()
+
+		//writing a suggestion to skip B4 end of fight.
+		if (this._missedF4s) {
+			this.suggestions.add(new Suggestion({
+				icon: ACTIONS.FIRE_IV.icon,
+				content: <Fragment>
+					You lost at least  one <ActionLink {...ACTIONS.FIRE_IV}/> by not skipping <ActionLink {...ACTIONS.BLIZZARD_IV}/> in the Umbral Ice phase before the fight finished.
+				</Fragment>,
+				severity: SEVERITY.MEDIUM,
+				why: <Fragment>
+					You missed {this._missedF4s} Fire IV{this._missedF4s > 1 && 's'}.
+				</Fragment>,
+			}))
+		}
+
+		//suggestion for unneccessary extra F1s.
+		if (this._extraF1s) {
+			this.suggestions.add(new Suggestion({
+				icon: ACTIONS.FIRE_I.icon,
+				content: <Fragment>
+					Casting more than one <ActionLink {...ACTIONS.FIRE_I}/> per Astral Fire cycle is a crutch that should be avoided by better pre-planning of the encounter.
+				</Fragment>,
+				severity: SEVERITY.MINOR,
```

Comment:
Is this _always_ going to be minor? What if I'm a total idiot and chain cast F1 in the fire cycle?

Reply:
good Idea. I'll add something